Configurations/WebKitProjectPaths.xcconfig

```diff
@@ -112,6 +112,16 @@ WEBCORE_LIBRARY_DIR[sdk=macosx*] = $(WEBCORE_LIBRARY_DIR_USE_OVERRIDE_FRAMEWORKS
 WEBCORE_LIBRARY_DIR_USE_OVERRIDE_FRAMEWORKS_DIR_NO = $(UMBRELLA_FRAMEWORKS_DIR)/WebCore.framework/Versions/A/Frameworks;
 WEBCORE_LIBRARY_DIR_USE_OVERRIDE_FRAMEWORKS_DIR_YES = $(WK_OVERRIDE_FRAMEWORKS_DIR);
 
+WEBKITLEGACY_PRIVATE_HEADERS_DIR = $(WEBKITLEGACY_PRIVATE_HEADERS_DIR_$(CONFIGURATION));
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Release = $(WEBKITLEGACY_PRIVATE_HEADERS_engineering);
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Debug = $(WEBKITLEGACY_PRIVATE_HEADERS_engineering);
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production = $(SDK_DIR)$(WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_COCOA_TOUCH_$(WK_IS_COCOA_TOUCH));
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_COCOA_TOUCH_YES = $(PRODUCTION_PRIVATE_FRAMEWORKS_DIR)/WebKitLegacy.framework/$(WK_FRAMEWORK_VERSION_PREFIX)PrivateHeaders;
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_COCOA_TOUCH_NO = $(WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_$(WK_DEFAULT_PLATFORM_NAME)_USE_OVERRIDE_FRAMEWORKS_DIR_$(WK_USE_OVERRIDE_FRAMEWORKS_DIR));
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_macosx_USE_OVERRIDE_FRAMEWORKS_DIR_NO = $(UMBRELLA_FRAMEWORKS_DIR)/WebKitLegacy.framework/$(WK_FRAMEWORK_VERSION_PREFIX)PrivateHeaders;
+WEBKITLEGACY_PRIVATE_HEADERS_DIR_Production_macosx_USE_OVERRIDE_FRAMEWORKS_DIR_YES = $(WK_OVERRIDE_FRAMEWORKS_DIR)/WebKitLegacy.framework/$(WK_FRAMEWORK_VERSION_PREFIX)PrivateHeaders;
+WEBKITLEGACY_PRIVATE_HEADERS_engineering = $(BUILT_PRODUCTS_DIR)/WebKitLegacy.framework/$(WK_FRAMEWORK_VERSION_PREFIX)PrivateHeaders;
```

Comment:
There is no mistake here, but should this be renamed to `WEBKITLEGACY_PRIVATE_HEADERS_DIR_engineering` for consistency?

Reply:
The lines I based this off of do the same thing, but I agree that it's oddly named. Will update them all.